datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/config/AppConfigResolver.java

```diff
@@ -81,7 +84,8 @@ public AppConfigResolver(
       final SearchBarConfiguration searchBarConfig,
       final HomePageConfiguration homePageConfig,
       final FeatureFlags featureFlags,
-      final ChromeExtensionConfiguration chromeExtensionConfiguration) {
+      final ChromeExtensionConfiguration chromeExtensionConfiguration,
+      final SettingsService settingsService) {
```

Comment:
historically - all settings stored in the Settings service were fetched using standalone graphql queries for each setting category. That is an antipattern, because it meant a linearly increasing number of HTTP calls, one per setting. I am breaking out of the pattern here and including my setting in the AppConfig, which we already fetch on every page load.

As a result, settingsService now becomes a dependency for AppConfigResolver.